### PR TITLE
NETOBSERV-1792: Added update-build script to add downstream labels

### PR DESCRIPTION
**Pull request by OlivierCazade**
## Description

Added update-build script to add downstream labels

## Dependencies

n/a

## Checklist

If you are not familiar with our processes or don't know what to answer in the list below, let us know in a comment: the maintainers will take care of that.

* [X] Will this change affect NetObserv / Network Observability operator? If not, you can ignore the rest of this checklist.
* [ ] Is this PR backed with a JIRA ticket? If so, make sure it is written as a title prefix _(in general, PRs affecting the NetObserv/Network Observability product should be backed with a JIRA ticket - especially if they bring user facing changes)._
* [ ] Does this PR require product documentation?
  * [ ] If so, make sure the JIRA epic is labelled with "documentation" and provides a description relevant for doc writers, such as use cases or scenarios. Any required step to activate or configure the feature should be documented there, such as new CRD knobs.
* [ ] Does this PR require a product release notes entry?
  * [ ] If so, fill in "Release Note Text" in the JIRA.
* [ ] Is there anything else the QE team should know before testing? E.g: configuration changes, environment setup, etc.
  * [ ] If so, make sure it is described in the JIRA ticket.
* QE requirements (check 1 from the list):
  * [ ] Standard QE validation, with pre-merge tests unless stated otherwise.
  * [ ] Regression tests only (e.g. refactoring with no user-facing change).
  * [ ] No QE (e.g. trivial change with high reviewer's confidence, or per agreement with the QE team).

### CODE REVIEW

File: hack/update-build.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+: "${CONTAINER_FILE:=./Dockerfile}"
+: "${COMMIT:=$(git rev-list --abbrev-commit --tags --max-count=1)}"
+
+cat <<EOF >>"${CONTAINER_FILE}"
+LABEL com.redhat.component="network-observability-ebpf-agent-container"
+LABEL name="network-observability-ebpf-agent"
+LABEL io.k8s.display-name="Network Observability EBPF Agent"
+LABEL io.k8s.description="Network Observability EBPF Agent"
+LABEL summary="Network Observability EBPF Agent"
```
Comment (jotak) [inline hack/update-build.sh:13]: nit: `eBPF` with low-case "e" is the conventional writing

```suggestion
LABEL io.k8s.display-name="Network Observability eBPF Agent"
LABEL io.k8s.description="Network Observability eBPF Agent"
LABEL summary="Network Observability eBPF Agent"
```

Comment (OlivierCazade) [reply hack/update-build.sh]: Thanks!

**Pull request merged**